docs/users/web3.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Use Clerk with Web3
+description: Learn to install and initialize Clerk in a new Web3 application.
+---
+
+# Use Clerk with Web3
+
+Learn how to use Clerk to quickly and easily add secure authentication and user management to your Web3 application.
+
+## Before you start
+
+You need to create a Clerk application in your [Clerk Dashboard](https://dashboard.clerk.com/) and configure the application to use **Sign in with Metamask**. For more information on how to set up a Clerk application, check out the [setup guide](/docs/quickstarts/setup-clerk). To enable Metamask authentication, check out [this guide](/docs/authentication/configuration/sign-in-options#social-o-auth-web3).
```

Comment:
The anchor contains the phrase `social-o-auth`. It needs to be replaced with `web3-metamask` as this is not a social login connection like Google, or Facebook.

Reply:
It's because it's linking to a section that talks about adding social oauth *plus* web3 auth. I can remove this link altogether and lay out the instructions (to adding web3 auth) here instead, if that feels more accurate/helpful.